app/javascript/packs/hero-rotation.js

```diff
@@ -32,25 +38,40 @@ window.addEventListener('load', () => {
 
   const renderNextImage = images => {
     const [activeElem, inactiveElem] = getImageElements();
-    const nextImage = images[getNextImage(images, activeElem.style.backgroundImage)];
+    const nextImage =
+      images[getNextImage(images, activeElem.style.backgroundImage)];
```

Comment:
This was because I ran prettier on the file and it had a line length violation. Not a major deal overall but also an automated style choice as opposed to an active one.